examples/next/pages/ui/components/authenticator/auth-with-multi-alias/index.page.tsx

```diff
@@ -0,0 +1,18 @@
+import { Authenticator } from '@aws-amplify/ui-react';
+import { Amplify } from 'aws-amplify';
+import awsExports from '@environments/auth-with-multi-alias/src/aws-exports';
+
+Amplify.configure({
+  ...awsExports,
+  auth: {
+    login_mechanisms: ['username', 'email', 'phone_number'],
```

Comment:
We really need that `amplify pull` update to land so we can see what _actually_ comes back.

Reply:
Definitely. I take some comfort in the fact that our UI will automatically handle whatever login mechanisms are returned. We'll just need to make sure that the login mechanisms consumed as part of zero-config match what's actually possible in Cognito. I feel it best we provide guide rails for customers as much as possible.